services/QuillLMS/engines/evidence/app/controllers/evidence/activities_controller.rb

```diff
@@ -91,6 +91,8 @@ def activity_versions
       render json: @activity&.activity_versions
     end
 
+    EXAMPLES_KEY = Evidence::Synthetic::SeedDataGenerator::EXAMPLES_KEY
```

Comment:
I don't think this is being used, is it?

Reply:
Ah, good call, that was from an earlier iteration, will remove.